src/chrome/content/rules/A1-Telekom-Austria.xml

```diff
@@ -34,7 +34,6 @@
 		to="https://$1a1community.net/" />
 
 	<!--	!www doesn't work over https.	-->
```

Comment:
This comment is now floating. Move it to the head of the file and indicate 'www.telekom.at' rather than just www.